src/js/component/RegisterModal.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect, useContext } from "react";
+import { Context } from "../store/appContext";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import "../../styles/registermodal.scss";
+import logo2 from "../../img/logo2.jpg";
+import profilephoto from "../../img/profilephoto.jpg";
+
+export const Registermodal = () => {
+	const [show, setShow] = useState(false);
+	const handleClose = () => setShow(false);
+	const handleShow = () => setShow(true);
+	const { store, actions } = useContext(Context);
+	const [username, setUsername] = useState("");
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [location, setLocation] = useState("");
+	const [error, setError] = useState("");
+	const [displayerror, setDisplayerror] = useState("Errormesage");
```

Comment:
Pon en orden esta lista de variables por fa, que se mezclan los los useState con el Context

Reply:
Podéis agrupar todo en un objeto